examples/cpp/raw_mandelbrot.cpp

```diff
@@ -0,0 +1,175 @@
+#include <stdlib.h>
```

Comment:
Prefer `#include <cstdlib>` - being able to use the .h form from C is incidental behaviour that may be patched out the standard or warned upon in the future. Tools such as `clang-tidy` already issue a warning about this.

Reply:
You are right, it seems it was deprecated long ago but kept in order to allow compatibility. 
Also, cstdlib uses namespace so you can keep yours cleaner.
There are some uses of stdlib.h spread around fract4dc classes in the codebase, I will try to change those too.